portal/models/reporting.py

```diff
@@ -0,0 +1,81 @@
+"""Reporting statistics and data module"""
+from collections import defaultdict
+
+from ..dogpile import dogpile_cache
+from .fhir import CC
+from .intervention import Intervention
+from .procedure_codes import known_treatment_started
+from .procedure_codes import known_treatment_not_started
+from .role import ROLE
+from .user import User
+
+
+@dogpile_cache.region('hourly')
+def get_reporting_stats():
+    """Cachable interface for expensive reporting data queries
+
+    The following code is only run on a cache miss.
```

Comment:
I know there's logging in the celery task, but I think it'd be good to note this is running (catch any overlooked cache misses).